src/diffusers/models/attention.py

```diff
@@ -243,6 +243,8 @@ class BasicTransformerBlock(nn.Module):
             The type of positional embeddings to apply to.
         num_positional_embeddings (`int`, *optional*, defaults to `None`):
             The maximum number of positional embeddings to apply.
+        squeeze_hidden_states (`bool`, *optional*, defaults to `True`): 
```

Comment:
I think this needs to be removed? Argument doesn't seemed to be used.

Reply:
Okay. I have removed it.